lib/createDist.js

```diff
@@ -19,6 +19,9 @@ const createDist = (buildConfig = config.defaultBuildConfig, options) => {
   fs.removeSync(path.join(config.outputDir, 'dist'))
   config.buildTarget = 'create_dist'
   util.buildTarget()
+  if (config.targetOS === 'android') {
```

Comment:
this should still have `config.shouldSign()` in addition to the targetOS check

Reply:
we can add it as a follow up, as should be applied in any case on all 3 branches. But I think we shouldn't skip signing Android apks. First it's fast, second we will need that on tests to check upgrade scenarios.